### PR TITLE
rescale actb_incidence_count to national level

### PR DESCRIPTION
**Pull request by yongqx**
The `actb_incidence_count` has been rescaled to national level. To be clear:
scaled_actb_incidence_count = (actb_incidence_rate * GBD 2017 age-/sex- specific national population) / 100_000

### CODE REVIEW

**Comment by aflaxman** [comment]
This looks like what I was expecting.  If you are able to run it, can you paste the results in here for me to review?  Does a spot check of the count compared to the population and the rate make sense in a few purposefully selected spots?

**Comment by yongqx** [comment]
> This looks like what I was expecting. If you are able to run it, can you paste the results in here for me to review? Does a spot check of the count compared to the population and the rate make sense in a few purposefully selected spots?

I'll try run it and see if results being updated. I believe @halfhorst is working on incorporate the changes and make tool to allow us run in one command line.

**Comment by halfhorst** [comment]
incorporating this now.

edit: it has been worked in, see #98

File: src/vivarium_csu_ltbi/tools/ltbi_results_postprocessing.py [hunk 1]
```diff
@@ -90,6 +144,50 @@ def get_person_time(data):
...
+    pt_u5_hhtb = (pt
+                  .loc[pt.risk_group == 'u5_hhtb']
+                  .drop(columns='risk_group')
+                  .set_index(pt_cols))
+    pt_all_population = (pt
+                         .loc[pt.risk_group == 'all_population']
+                         .drop(columns='risk_group')
+                         .set_index(pt_cols))
+    
+    plwhiv_prop = pt_plwhiv / pt_all_population
```
Comment (yaqi7) [inline src/vivarium_csu_ltbi/tools/ltbi_results_postprocessing.py:173]: For calculating proportion, do you use person-time unit or count unit?

Comment (yongqx) [reply src/vivarium_csu_ltbi/tools/ltbi_results_postprocessing.py:173]: I used person time measure from sim output.

**Comment by halfhorst** [comment]
Hey @yongqx , you wanna take a stab at making these edits to the results processing tool? 
see here https://github.com/ihmeuw/vivarium_csu_ltbi/blob/master/src/vivarium_csu_ltbi/results_processing/counts_output.py#L158

This is the tool you can run with `make_results`

File: src/vivarium_csu_ltbi/tools/ltbi_results_postprocessing.py (hunk 1 above)
Comment (yaqi7) [reply src/vivarium_csu_ltbi/tools/ltbi_results_postprocessing.py:173]: Thank you for confirming it. For calculating U5 HH AcTB exposure proportion, I used counts of U5 with HH TB exposure/ counts of total U5 children and the counts are for all years (from start year to 2020). @aflaxman Do you think I need to update my script to make it consistent with Yongquan's .

Comment (yongqx) [reply src/vivarium_csu_ltbi/tools/ltbi_results_postprocessing.py:173]: I think the person-years measure is a count space metric.

**Comment by yongqx** [comment]
> Hey @yongqx , you wanna take a stab at making these edits to the results processing tool?
> see here https://github.com/ihmeuw/vivarium_csu_ltbi/blob/master/src/vivarium_csu_ltbi/results_processing/counts_output.py#L158
> 
> This is the tool you can run with `make_results`

Do i need to add `get_risk_specific_population` to that .py? The `get_national_population` become part of the process when output national population by risk group.

**Comment by yongqx** [comment]
@halfhorst Have you incorporated the line 484-501 in the run tool? This allows you add up two sim age end results and stack all countries measure into a giant output table shell.

**Pull request closed without merging**